src/core_plugins/vega/public/data_model/vega_parser.js

```diff
@@ -294,7 +331,9 @@ export class VegaParser {
     // `false` is a valid value
     res.mapStyle = this._config.mapStyle === undefined ? `default` : this._config.mapStyle;
     if (res.mapStyle !== `default` && res.mapStyle !== false) {
-      this._onWarning(`config.kibana.mapStyle may either be false or "default"`);
+      this._onWarning(i18n.translate('vega.vegaParser.mapStyleValueTypeWarningMessage', {
+        defaultMessage: 'config.kibana.mapStyle may either be false or "default"',
```

Comment:
Let's separate config name `config.kibana.mapStyle` into `values` since it should not be translated.